### PR TITLE
Update code documentation

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

**Comment by SpacePossum** [comment]
:+1:  looks good :)

File: Symfony/CS/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ class FixCommand extends Command
     /**
      * Config instance.
```
Comment (SpacePossum) [inline Symfony/CS/Console/Command/FixCommand.php]: class implementing `ConfigInterface` ?

Comment (keradus) [inline Symfony/CS/Console/Command/FixCommand.php]: it's not a type of content but content itself described here.
Therefore sth like `instance of class implementing ConfigInterface`... so long ;)

Comment (SpacePossum) [inline Symfony/CS/Console/Command/FixCommand.php]: fair enough :)

**Pull request merged**